README.md

```diff
@@ -301,51 +299,50 @@ This object contains the results of the simulation and provides insights into va
-  - **Element Type**: Float
-  - **Length**: 35
-- **Eigenverbrauch_Wh_pro_Stunde**: An array of floats representing the simulated self-consumption in watt-hours per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
-- **Einnahmen_Euro_pro_Stunde**: An array of floats representing the simulated income in euros per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
-- **Gesamt_Verluste**: The total simulated losses in watt-hours.
-  - **Type**: Float
-- **Gesamtbilanz_Euro**: The total simulated balance in euros.
-  - **Type**: Float
-- **Gesamteinnahmen_Euro**: The total simulated income in euros.
-  - **Type**: Float
-- **Gesamtkosten_Euro**: The total simulated costs in euros.
-  - **Type**: Float
-- **Haushaltsgeraet_wh_pro_stunde**: An array of floats representing the simulated energy consumption of a household appliance in watt-hours per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
-- **Kosten_Euro_pro_Stunde**: An array of floats representing the simulated costs in euros per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
-- **Netzbezug_Wh_pro_Stunde**: An array of floats representing the simulated grid consumption in watt-hours per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
-- **Netzeinspeisung_Wh_pro_Stunde**: An array of floats representing the simulated grid feed-in in watt-hours per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
-- **Verluste_Pro_Stunde**: An array of floats representing the simulated losses per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
-- **akku_soc_pro_stunde**: An array of floats representing the simulated state of charge of the battery in percentage per hour.
-  - **Type**: Array
-  - **Element Type**: Float
-  - **Length**: 35
+  - **E-Auto_SoC_pro_Stunde**: An array of floats representing the simulated state of charge of the electric car per hour.
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Eigenverbrauch_Wh_pro_Stunde**: An array of floats representing the simulated self-consumption in watt-hours per hour.
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Einnahmen_Euro_pro_Stunde**: An array of floats representing the simulated income in euros per hour.
+    - **Type**: Array
+    - **Element Type**: Float
+    - **Length**: 35
+  - **Gesamt_Verluste**: The total simulated losses in watt-hours.
+    - **Type**: Float
+  - **Gesamtbilanz_Euro**: The total simulated balance in euros.
+    - **Type**: Float
+  - **Gesamteinnahmen_Euro**: The total simulated income in euros.
+    - **Type**: Float
+  - **Gesamtkosten_Euro**: The total simulated costs in euros.
+    - **Type**: Float
+  - **Home_appliance_wh_per_hour**: An array of floats representing the simulated energy consumption of a household appliance in watt-hours per hour.
```

Comment:
simulated_consumption_home_appliances_Wh_per_hour

(bits are cheap, these days ;))

Reply:
done. reworked in #204